modules/tensor_mechanics/include/bcs/PenaltyInclinedBC.h

```diff
@@ -0,0 +1,45 @@
+//* This file is part of the MOOSE framework
```

Comment:
Since this is mechanics-specific, maybe this should be named something like `PenaltyInclinedNoDisplacementBC`?